csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -341,28 +362,50 @@ cdef class Integer(Number):
         import sympy
         return sympy.Integer(deref(self.thisptr).__str__().decode("utf-8"))
 
+    def _sage_(self):
+        IF HAVE_SAGE == True:
```

Comment:
This depends on whether we have the Sage libraries at compile time since I am importing Sage's cython version.